CMakeLists.txt

```diff
@@ -153,6 +153,26 @@ enable_cppcheck()
 # Pull in helper macros for working with leatherman libraries
 include(leatherman)
 
+# Guard to make sure we don't compile what we don't have in Leatherman
```

Comment:
why is guard enabled only in Windows?

Reply:
we only include the source file if `if ("${CMAKE_SYSTEM_NAME}" MATCHES "Windows")`, so it's redundant to have it on other platforms